docs/ai-helper/context-templates.md

```
@@ -0,0 +1,235 @@
+import { ContractValidator } from '@cvt/cvt-sdk';
+
+const validator = new ContractValidator('localhost:9550');
+await validator.registerSchema('api-name', schemaContent);
+
+const result = await validator.validate(
+  { method: 'GET', path: '/users/123' },
+  { statusCode: 200, body: '{"id": "123"}' }
+);
+```
+
+### Python SDK
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)